pkg/infra/tracing/tracing.go

```diff
@@ -79,6 +87,16 @@ func (ts *TracingService) initGlobalTracer() error {
 		options = append(options, jaegercfg.Tag(tag, value))
 	}
 
+	if ts.useZipkinPropagation {
```

Comment:
If this is all that is required to pass/start new spans for zipkin I'm more then happy to support it :)

Reply:
Yup, this is all that's needed.  You guys did a great job of instrumenting Grafana so these traces end up being super useful!